### PR TITLE
ignore early (before the stat) (HPC-7499)

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

File: lib/vsc/filesystem/posix.py [hunk 1]
```diff
@@ -257,16 +257,14 @@ def _local_filesystems(self):
             # returns [('rootfs', '/', 2051L, 'rootfs'), ('ext4', '/', 2051L, '/dev/root'),
             # ('tmpfs', '/dev', 17L, '/dev'), ...
             self.localfilesystemnaming = ['type', 'mountpoint', 'id', 'device']
+            # do we need further parsing, eg of autofs types or remove pseudo filesystems ?
+            if self.ignorefilesystems:
+                currentmounts = [x for x in currentmounts if not x[2] in OS_LINUX_IGNORE_FILESYSTEMS]
```
Comment (itkovian) [inline lib/vsc/filesystem/posix.py]: This seems odd. The comment above states that each `x` will be a tuple (type, mount point, id, device), so `x[2]` would be the ID, no? Maybe I am overlooking something :)

Comment (wdpypere) [reply lib/vsc/filesystem/posix.py]: yes, because the next line orders them in that way

Comment (wdpypere) [reply lib/vsc/filesystem/posix.py]: as you can see I now loop over currentmounts instead of self.localfilesystems

Comment (wdpypere) [reply lib/vsc/filesystem/posix.py]: only self.localfilesystems is in that particular order

Comment (itkovian) [reply lib/vsc/filesystem/posix.py]: Ah yes.

**Pull request merged**